services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -5969,7 +5969,7 @@ public VeniceControllerClusterConfig getControllerConfig(String clusterName) {
 
   /**
    * Validates that a store has been completely deleted from all venice clusters cross-regionally
-   * 
```

Comment:
nit: let's nix the extra line to keep things clean

Reply:
Done! Thanks!